CraftBukkit-Patches/0018-Fix-Dupe-Glitches-Including-Instamine.patch

```diff
@@ -0,0 +1,181 @@
+From 60e9199236ef93fd0f537a1eb976b2e151e65e96 Mon Sep 17 00:00:00 2001
+From: RhysB <jetpackingwolf@gmail.com>
+Date: Fri, 4 Oct 2019 23:41:14 +1000
+Subject: [PATCH] Fix Dupe Glitches Including Instamine
+
+
+diff --git a/src/main/java/net/minecraft/server/BlockChest.java b/src/main/java/net/minecraft/server/BlockChest.java
+new file mode 100644
+index 00000000..be79eb2b
+--- /dev/null
++++ b/src/main/java/net/minecraft/server/BlockChest.java
+@@ -0,0 +1,120 @@
++package net.minecraft.server;
++
++import java.util.Random;
++
++public class BlockChest extends BlockContainer {
```

Comment:
This should really be imported correctly using [importmcdev.sh](https://github.com/canyonmodded/canyon/blob/master/scripts/importmcdev.sh).